### ISSUE TITLE
How many iterations or loss, can I have a so-so results on my test images ?

### ISSUE THREAD

**Issue by XiaoLaoDi** [clovaai/deep-text-recognition-benchmark#14]

**Comment by XiaoLaoDi** [comment]
I have trained the model about 12000 iterations, and the loss become more and more small (valid loss is nearly 0.34375, and best_accuracy is nearly 80.332), but when i test the model with the best_accuracy.pth model, the results seems so bad, just likely trained nothing. So the problem is how many iterations of loss, can i have a soso results ?

**Comment by XiaoLaoDi** [comment]
I trained the TPS-ResNet-BiLSTM-Attn with PAD on.

**Comment by ku21fan** [comment]
Hello, 

1. 'how many iterations of loss, can I have a soso results?'
In our case, the accuracy on valid data is over 70% with about 10000 iterations, and start to get so-so result.
Thus, you can roughly judge your model at 10000 iterations.
I attached the train log our pretrained model (case insensitive), hope it helps you.
[log_train.txt](https://github.com/clovaai/deep-text-recognition-benchmark/files/3304492/log_train.txt)

2. 'but when i test the model with the best_accuracy.pth model, the results seems so bad, just likely trained nothing.'
One possible reason is that If you do `--PAD` option only when training part, the result would be bad. 
If you trained the model with `--PAD` option, you should also test with `--PAD` option. 
Please check the test accuracy with `--PAD` option.

Best,
Baek.

**Comment by XiaoLaoDi** [comment]
@ku21fan thanks for your reply. Maybe my test data is not draw from iid dataset, I just randomly cut some images to test my model, which is not the same source where my training data coming from.I trained the model with much more Chinese characters sequence image(with little english sequence images), but i test the model in totally english sequence images, so maybe that's the problem.

**Comment by XiaoLaoDi** [comment]
I have just tested the model with some sequence images draw from the same iid dataset, and the results seems ok.

**Comment by cwellszhang** [comment]
@ku21fan I have trained the model about 10000 iterations for many times but the best_accuracy is always >2% lower than 70% (about 65~68% ) even though I use the same training setting as yours. What's the problem behind this? am l missing something?

**Comment by ku21fan** [comment]
@cwellszhang
The accuracy at the beginning would be influenced by the initialization.
Thus, try different seed option with `--manualSeed 1113` or `--manualSeed 2222`.

**Comment by cwellszhang** [comment]
thanks a lot. I have figured out the problem.
